src/project_generator.rs

```diff
@@ -49,6 +64,16 @@ impl Generator {
         let content = self.get_problem_content().await?;
 
         html::generate_markdown(self.project_title.clone(), &content, self.directory.clone())?;
+
+        println!(
+            "{}",
+            format!(
+                "🌟 Leetcode project was created in dir {} with language {}",
+                &self.directory.bold(),
+                &self.config.default_lang
+            ).green()
+        );
+
```

Comment:
I'm okay with this, but when creating a project for rust with `Command::new("cargo").arg("new")...` it already prints an output `Created binary (application)...`. Would be good to suppress that message so that we don't get output 2 times.

Again, I think using `log::info` would be better here, because it allows the user to choose if he wants to see any output with `env_logger` instead of forcing the print.